common/v2/components/InputField.tsx

```diff
@@ -114,17 +134,21 @@ export class InputField extends Component<Props> {
             <CustomTextArea
               value={value}
               onChange={onChange}
+              onBlur={onBlur}
               inputError={inputError}
               onKeyUp={this.handleKeyUp}
+              placeholder={placeholder ? placeholder : ''}
```

Comment:
Since his ternary is repeated, how about:
```
const {
  placeholder = ''
} = this.props

... placeholder={placeholder} >
```

Reply:
Makes sense, fixed.